index.html

```diff
@@ -8,44 +8,273 @@
+                            </p>
+                        </div>
+                    </div>
+                    <p class="works-item-project-description"> A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="works-item-project-tecnologies">
+                        <li class="works-item-project-tecnologies-item works-item-project-item-html"> 
+                            <span> html </span> 
+                        </li>
+                        <li class="works-item-project-tecnologies-item works-item-project-item-css"> 
+                            <span> css </span>
+                        </li>
+                        <li class="works-item-project-tecnologies-item works-item-project-item-javaScript"> 
+                            <span> javaScript </span>
+                        </li>
+                    </ul>
+                    <div id="works-item-project-action">
+                        <button class="works-item-project-action-button">
+                            <span>  See Project </span>
+                        </button>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section id="about">
+            <div id="top">
+                <h2 class="about-title">About Myself</h2>
+                <p class="about-text"> Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <div class="about-social-media">
+                        <a class="supporting-text"">LET'S CONNECT</a>
+                    <ul class="about-social-media-links">
+                        <li>
+                            <a href="#">
+                                <img src="./img/twitter.png" alt="twitter link">
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#">
+                                <img src="./img/Linkedin.png" alt="Linkedin link">
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#">
+                                <img src="./img/m.png" alt="m link">
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#">
+                                <img src="./img/github.png" alt="twitter link">
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#">
+                                <img src="./img/m2.png" alt="m2 link">
+                            </a>
+                        </li>
+                    </ul>
+                </div>
+                <button id="btn-get-resume"><span> Get my resume</span></button>
```

Comment:
- [Optional] Please add a hover and pressed state to your buttons according to the design template. You can use the `:hover` and `:active` pseudo-classes to implement this states in your css.
![192084447-51f9e626-5b23-409a-aa19-dbb30b91ac33](https://user-images.githubusercontent.com/94551484/192089982-10df0c78-cd18-43c5-b7ee-e87794d9977b.png)